java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TextDocumentServiceImpl.java

```diff
@@ -1954,7 +1959,8 @@ CompletableFuture<List<Diagnostic>> computeDiagnostics(String uri, EnumSet<Error
     }
 
     private static final int DELAY = 500;
-
+    private static Pair<Boolean,String> hintsPrefsStatus = Pair.of(false,null);  
```

Comment:
Why should be the preference storage static ? Is there a benefit in having it as Pair<> insttead of two variables ?

Reply:
My thought process was to keep the variables which are related to each other in a pair, other then that there are no benefits of using Pair<> here. So, should I remove Pair and create two separate variables (I have no objection in that)?  
Regarding static part, I am really sorry I overlooked that, I would remove static and make updates accordingly.

Reply:
I have pushed the updated code.